test/prog/dftb+/timeprop/CH3_dftbu_singlet/dftb_in.hsd

```diff
@@ -70,10 +70,8 @@ ParserOptions = {
   ParserVersion = 3
```

Comment:
Should this still be version 3?

Reply:
Unfortunately not, as the finite difference methods changed after that parser version (this could either be requested with an extra keyword, or the test recomputed with the more accurate new default).

Reply:
Replaced it with a recomputed 20.1 input.